### ISSUE TITLE
Do not extract to /.fr-XXXXXX

### ISSUE THREAD

**Issue by GreenLunar** [mate-desktop/engrampa#83]
Upon opening file(s) directly from Engrampa, extract to /tmp/eng-XXXXXX (Xarchiver style) or /tmp/engrampa.XXXXXX (Comix style).

**Comment by monsta** [comment]
Looks like it will reintroduce [this bug](https://bugzilla.gnome.org/show_bug.cgi?id=303213). Any good reasons?

**Comment by GreenLunar** [comment]
Yes, because there are times when files are extracted to `~/Documents/Events/.fr-XXXXXX`.

I do not know why it occurs, but I guess*  it happens when working directory is not /home/username/ (e.g. when executing in a folder from terminal).

`* Even though this guess is unlikely, it is the best I could think of`
`* I find out about it days, weeks and even months after it occurs`

**Comment by GreenLunar** [comment]
> I do not know why it occurs, but I guess* it happens when working directory is not /home/USER_NAME/ (e.g. when executing in a folder from terminal).

Try drag-n-drop content of archive and you might see a redundant (hidden) folder in the source or destination directory.
